Client/Frontend/Theme/LightTheme.swift

```diff
@@ -14,34 +14,54 @@ struct LightTheme: Theme {
 }
```

Comment:
nit: on top of the LightTheme struct the comment should be updated or removed as it mentions fonts

Reply:
I opted to remove both because they don't make a whole lot of sense anymore.